pallets/subtensor/src/lib.rs

```diff
@@ -816,6 +852,14 @@ pub mod pallet {
     pub type SenateRequiredStakePercentage<T> =
         StorageValue<_, u64, ValueQuery, DefaultSenateRequiredStakePercentage<T>>;
 
+    #[pallet::storage]
+    pub type StakeJobs<T: Config> =
+        StorageMap<_, Blake2_128Concat, u64, StakeJob<T::AccountId>, OptionQuery>;
```

Comment:
Just a curiosity for myself but I tend to use non-cryptographic hash (XXHash) or even Identity (raw) for on chain incremented index, given it's under chain control and not provided by the user, any idea which one is better?

Reply:
This article seems like a good answer: https://www.shawntabrizi.com/blog/substrate/transparent-keys-in-substrate/